### PR TITLE
Create brand-guildlines.md

### PR DESCRIPTION
**Pull request by Lykosteng**
add brand-guildlines.md

### CODE REVIEW

**Comment by Lykosteng** [comment]
@lilin90 test

File: logos/brand-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Resources and Guidelines for TiDB Community
+
+These resources and guidelines were designed to assist the TiDB Community to effectively and consistently build and protect the community’s brand 
+through visual communication.
```
Comment (lilin90) [inline logos/brand-guidelines.md]: Change to:
The following resources and guidelines aim to assist ... the community's brand ...

And pay attention to the punctuation.

File: logos/brand-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+# TiDB Logo
+
+## Seal+Signature(formal)
```
Comment (lilin90) [inline logos/brand-guidelines.md]: Leave a whitespace before the parenthesis.

Pls also update the following ones.

File: logos/brand-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+CMYK: 0-0-0-100 ; RGB: 0-0-0 ; Hex: 000000
+
+# Do these awesome things
+
+- √ Use the TiDB logo to link to TiDB and PingCAP
+- √ Use the Mark in social buttons to link to your TiDB project
+- √ Use the TiDB logo to advertise that your product use TiDB as servise
+- √ Use the TiDB logo in a blog post or news article about TiDB
+
+# Please don’t do these things
```
Comment (lilin90) [inline logos/brand-guidelines.md]: I think it'd be more clear to change to:

# Never do these things

File: logos/brand-guidelines.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+**Cool Grey**
+CMYK: 48-36-24-66 ; RGB: 77-79-83 ; Hex: 4D4F53
+
+**Black**
+CMYK: 0-0-0-100 ; RGB: 0-0-0 ; Hex: 000000
+
+# Do these awesome things
+
+- √ Use the TiDB logo to link to TiDB and PingCAP
+- √ Use the Mark in social buttons to link to your TiDB project
```
Comment (lilin90) [inline logos/brand-guidelines.md]: If the "Mark" here means the same with the "TiDB logo", then keep consistent.

File: logos/brand-guidelines.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+- √ Use the TiDB logo to advertise that your product use TiDB as servise
```
Comment (lilin90) [inline logos/brand-guidelines.md]: -> Use the TiDB logo to show that you use the TiDB services

File: logos/brand-guidelines.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+- √ Use the TiDB logo in a blog post or news article about TiDB
+
+# Please don’t do these things
+
+- X Use the TiDB logo for your application’s icon
```
Comment (lilin90) [inline logos/brand-guidelines.md]: -> Use the TiDB logo as your application's icon

Comment (Lykosteng) [reply logos/brand-guidelines.md]: thanks a lot

**Comment by Lykosteng** [comment]
@shenli  PTAL

**Comment by shenli** [comment]
Close it due to stale

**Pull request closed without merging**